maint/GenerateTest26.py

```diff
@@ -55,7 +55,7 @@ def gen_script_tests():
   script_data = [None] * len(script_names)
   char_data = [None] * 0x110000
 
-  property_re = re.compile("^([0-9A-F]{4,6})(?:\\.\\.([0-9A-F]{4,6}))? +; ([A-Za-z_ ]+) #")
+  property_re = re.compile("^([0-9A-F]{4,6})(?:\\.\\.([0-9A-F]{4,6}))? +; ([A-Za-z_ ]+[A-Za-z]) +#")
```

Comment:
it seems that the original code included spaces by mistake and should had been `([A-Za-z_]+)`, why was the expression changed?

Reply:
The format went from something like this
```
A9CF          ; Bugi Java # Lm       JAVANESE PANGRANGKEP
```

to:

```
A9CF          ; Bugi Java                      # Lm      JAVANESE PANGRANGKEP
```

The previous expression's `[A-Za-z_ ]+` capture-group matches the space between `Bugi` and `Java` for example, as well as all the trailing spaces except for the last one. So the `split(" ")` call was thrown off. I adjusted it to capture all in-between spaces, but not the trailing spaces used in the UCD-16's for padding.

Reply:
The point I was trying to make is that the same code change is also most likely needed in the other places where this file is processed.

Reply:
Thank you, you are right, there was indeed one instance of `ScriptExtensions.txt` parsing in the `GenerateCommon.py` script that had a similar regex as well, which I adjusted and force-pushed.